src/QMCApp/QMCMain.cpp

```diff
@@ -124,18 +124,17 @@ QMCMain::QMCMain(Communicate* c)
 
   // Record features configured in cmake or selected via command-line arguments to the printout
   app_summary() << std::endl;
-#if !defined(ENABLE_OFFLOAD) && !defined(ENABLE_CUDA) && !defined(QMC_CUDA) && !defined(ENABLE_ROCM)
+#if !defined(ENABLE_OFFLOAD) && !defined(ENABLE_CUDA) && !defined(QMC_CUDA) && !defined(ENABLE_HIP)
   app_summary() << "  CPU only build" << std::endl;
 #else
 #if defined(ENABLE_OFFLOAD)
   app_summary() << "  OpenMP target offload to accelerators build option is enabled" << std::endl;
 #endif
-#if defined(ENABLE_CUDA) || defined(QMC_CUDA)
+#if defined(ENABLE_HIP) || defined(QMC_CUDA2HIP)
+  app_summary() << "  HIP acceleration build option is enabled" << std::endl;
+#elif defined(ENABLE_CUDA) || defined(QMC_CUDA)
```

Comment:
I think it would be better if these had distinct outputs
i.e. 
legacy CUDA acceleration build option is enabled
CUDA acceleration build option is enabled

Reply:
I think it is better to expand all cases to be accurate.